heat/core/_operations.py

```diff
@@ -422,7 +422,7 @@ def __reduce_op(
     balanced = x.balanced
 
     # if local tensor is empty, replace it with the identity element
-    if 0 in x.lshape and (axis is None or (x.split in axis)):
+    if split is not None and 0 in x.lshape and (axis is None or split in axis):
```

Comment:
```suggestion
    if x.is_distributed() and 0 in x.lshape and (axis is None or split in axis):
```
A small suggestion: x.is_distributed() also checks the split